### ISSUE TITLE
Poor accessibility for new students who want to contribute

### ISSUE THREAD

**Issue by LuckyLolo123** [freeCodeCamp/freeCodeCamp#45179]
## Is your feature request related to a problem? Please describe.
When I started the first curriculum of freecodecamp (beta) I found an error, but I didn't care about it, after that error I found others and I decided to search through the FCC webpage how to report it, It took me a while to find how to report an error. Right now I have the link saved in favorite, and I have to look for it every time I want to report an issue with the curriculum. So, I think this is a barrier for people who want to contribute.

## Describe the solution you'd like
I would like to see a big, red, contrastable, and clickable button saying something like "Report an issue", where you can click on and take you directly to this link (https://github.com/freeCodeCamp/freeCodeCamp/issues/new/choose) where you can report the issue you found.

**Comment by LuckyLolo123** [comment]
thanks for the quick response, that makes sense.

**Comment by ieahleen** [comment]
the footer of each freecodecamp page has a link to the github repo, it's the "open source" link

**Comment by LuckyLolo123** [comment]
> the footer of each freecodecamp page has a link to the github repo, it's the "open source" link

I couldn't find that link. Besides that @moT01 is right, I think this should stay as is.

**Comment by jeremylt** [comment]
I agree with the comments so far - the forum should stay the first line of support!

**Comment by ieahleen** [comment]
> > the footer of each freecodecamp page has a link to the github repo, it's the "open source" link
> 
> I couldn't find that link. Besides that @moT01 is right, I think this should stay as is.

@LuckyLolo123 

![image](https://user-images.githubusercontent.com/26656284/154914632-b9044bd3-80c8-4f2e-8d00-dbfaa5ba91cf.png)
